src/app/helptext/sharing/smb/smb.ts

```diff
@@ -104,6 +101,29 @@ export const helptext_sharing_smb = {
     tooltip_auxsmbconf: T('Additional <b>smb5.conf</b> parameters not covered by\
  other option fields.'),
 
+    placeholder_aapl_name_mangling: T('Enable Apple-style Character'),
+    tooltip_aapl_name_mangling: T('Enable Apple-style character encoding for NTFS illegal characters.'),
+
+    placeholder_streams: T('Enable Alternate Data Streams'),
+    tooltip_streams: T('Enable support for Alternate Data Streams. Disabling support for streams will\
+ cause MacOS to write streams to files on the filesystem.'),
+
+    placeholder_durablehandle: T('Enable SMB2/3 durable handles'),
+    tooltip_durablehandle: T('Enable support for SMB2/3 durable handles. This parameter disables support\
+ for POSIX byte-range locks in Samba and may be unsuitable if multi-protocol (or local) access to the\
+ files is anticipated.'),
+
+    placeholder_fsrvp: T('Enable FSRVP'),
+    tooltip_fsrvp: T('Enable support for FSRVP. This protocol allows RPC clients to manage snapshots of the\
```

Comment:
Would it be helpful to users to spell out the acronym and make it a link to [Microsoft's documentation](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-fsrvp/dae107ec-8198-4778-a950-faa7edad125b)?
```
    tooltip_fsrvp: T('Enable support for the File Server Remote VSS Protocol (FSVRP). This protocol allows \
     RPC clients to manage snapshots of the ...
```